cumulus_etl/etl/studies/covid_symptom/covid_tasks.py

```diff
@@ -81,9 +81,7 @@ def is_ed_docref(docref) -> bool:
 class BaseCovidCtakesTask(tasks.BaseNlpTask):
     """Covid Symptom study task, to generate symptom lists from ED notes using cTAKES + a polarity check"""
 
-    tags: ClassVar = {"covid_symptom", "gpu"}
```

Comment:
This tags cleanup is unrelated - I just noticed it while adding new tasks.

I had made these tags obsolete when I added the `nlp` subtask and took out the no-longer-needed `--task-filter` support. So this is just some cleanup around that.